### ISSUE TITLE
Widget for Android

### ISSUE THREAD

**Issue by KushtrimPacaj** [devRant/devRant#63]
This is not a bug, but a feature proposal.

A widget for devRant for Android would be useful. Nothing top complicated, just a simple widget that shows one rant, and refreshes ( shows another one ) every few hours or so.

It'd allow us to read something funny (or sad), and laugh (or cry) without opening the app.

**Comment by m1guelpf** [comment]
> Nothing top complicated

NEVER SAY THAT. you sound like a client requesting "minor changes". Besides that, it's a great idea :wink:

**Comment by filthycoding** [comment]
It might be more complicated because the devRant app is actually not written in Java or Kotlin (refer to the about section in the app to see the technologies devRant builds upon)

**Comment by KushtrimPacaj** [comment]
Right, just searched for android widgets for Appcelerator Titanium, and it might not support it. This is why native rocks :/

**Comment by cozyplanes** [comment]
Just to say. [I dont think titanium support widgets.](https://jira.appcelerator.org/plugins/servlet/mobile#issue/TIMOB-1780) 
And as a side note, making widgets is a pain in the ass for devs
